subsystem/cluster_test.go

```diff
@@ -3168,76 +3167,6 @@ var _ = Describe("cluster install, with default network params", func() {
 		})
 	})
 })
-var _ = Describe("Update cluster with OLM operators", func() {
```

Comment:
Don't remove the entire function - only the part dealing with OCS

Reply:
@jakub-dzon suggested that we can remove the test as it's no longer valid

Reply:
Ack.